contracts/src/TokenHome/ERC20TokenHome.sol

```diff
@@ -174,7 +174,7 @@ contract ERC20TokenHome is IERC20TokenHome, TokenHome {
 
         // Reset the recipient contract allowance to 0.
         // Use of {safeApprove} is okay to reset the allowance to 0.
```

Comment:
Should this comment be updated to {forceApprove}?

Reply:
updated